docs/user/install.md

```diff
@@ -1,6 +1,6 @@
 **Note**: **_This installation method will be deprecated by the end of 2022.
 The current recommended method for installing Turbinia is
-[here](install-gke-pubsub.md)._**
+[here](https://github.com/google/osdfir-infrastructure/tree/main/charts/turbinia)._**
```

Comment:
Nitpick but can we point this to the main branch instead https://github.com/google/osdfir-infrastructure then there is a direct link there to the Turbinia helm chart. I did this with TS as well just way to get visibility into deploying an all in one OSDFIR Infra